Wrappers/Python/cil/utilities/display.py

```diff
@@ -96,42 +101,79 @@ def save(self,filename, **kwargs):
 
 class show1D(show_base):
     """
-    This creates 1D plots of pixel values by slicing multi-dimensional
-    data.
+    This creates and displays 1D plots of pixel values by slicing
+    multi-dimensional data.
+
+    The behaviour is as follows: if provided multiple datasets and a single
+    slice set (see first example below), one line plot will be generated
+    per dataset; if provided a single dataset and multiple sets of slices
+    (see second example below), one line plot will be generated per slice
+    set;  if provided multiple datasets and multiple slice sets, the
+    :math:`i`-th set of slices will apply to the :math:`i`-th dataset, with
+    a line plot generated in each case.
 
     Parameters
     ----------
     data : DataContainer, list of DataContainer, tuple of DataContainer
         Multi-dimensional data to be reduced to 1D.
-    line_coords : list of tuples, default=None
-        (dimension, coordinate) pairs for slicing `data`
+    line_coords : list of tuple or list of list of tuple, default=None
+        A list, or nested list, of (dimension, coordinate) pairs for
+        slicing `data` (default is None, which is only valid when 1D
+        data is passed)
     label : str, list of str, default=None
         Label(s) to use in the plot's legend
     title : str, default None
         A title for the plot
     color : str, list of str, default=None
         Color(s) for each line plot
+    ls : {"-","--","-.",":"}, dict of {"-","--","-.",":"}, default=None
+        Linestyle(s) for each line plot
+    axis_labels : tuple of str, list of str, default=('Pixel','Pixel value')
+        Axis labels in the form (x_axis_label,y_axis_label)
+    num_cols : int, default=3
+        The number of columns in the grid of subplots produced in the case
+        of multiple plots
     size : tuple, default=(8,6)
         The size of the figure
-    axis_labels : tuple of str, list of str, default=('Pixel','Pixel
-    value')
-        Axis labels in the form (x_axis_label,y_axis_label)
     force : bool, default=True
         Passed to `get_slice`
 
     Attributes
```

Comment:
This heading doesn't appear in the documentation:
![image](https://user-images.githubusercontent.com/60604372/216636887-fee4b856-f984-4ab1-b26a-944135fbd439.png)

Reply:
From looking here it seems like you did the right thing so it's odd: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Doesn't look like it would be an easy fix so don't worry about it if you can't fix it